rclcpp/include/rclcpp/node.hpp

```diff
@@ -279,6 +279,12 @@ class Node : public std::enable_shared_from_this<Node>
     const std::string & name,
     const ParameterT & value);
 
+  template<typename MapValueT>
```

Comment:
We should really have documentation for this method. I know it's a variant of another method that has no documentation, but I feel we should be moving toward better documentation with each new pr.

Reply:
Good idea.  Done in f039865cb0cf3b555f7aeefe15f85b4c2af67b12